vsclib/vsclib/vsc.py

```diff
@@ -0,0 +1,704 @@
+# (C) 2022 MBition GmbH
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+#
+# VSC Tree management
+#
+#  A namespace tree populated by data types, includes, and methods
+#
+NATIVE_DATATYPES = [ 'int8', 'uint8', 'int16', 'uint16', 'int32',
+                     'uint32', 'int64', 'uint64', 'float',
+                     'double', 'boolean', 'string', 'binary' ]
```

Comment:
`binary` supposed to be same as `byteBuffer` in https://github.com/COVESA/vehicle_service_catalog/blob/master/syntax.md#native-data-types?